advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -168,19 +171,30 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest? {
 }
 
 private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulnerability {
-    val (scoringSystem, severity) = severity.firstOrNull()?.let {
-        it.type.name to it.score
-    } ?: null to null
+    val (scoringSystem, score) = severity.firstOrNull()?.let {
+        val cvss = Cvss.fromVector(it.score) ?: run {
+            log.debug { "Could not parse CVSS vector from: '${it.score}'." }
+            return@let null to it.score
```

Comment:
This avoids the IMO not very readable `return@` construct:

        Cvss.fromVector(it.score)?.let { cvss ->
            val scoringSystem = when {
                // Work around for https://github.com/stevespringett/cvss-calculator/issues/56.
                it.score.startsWith("CVSS:") -> it.score.substringBefore("/")
                cvss is CvssV2 -> "CVSS:2.0"
                else -> cvss.vector.substringBefore("/", "CVSS")
            }

            scoringSystem to cvss.calculateScore().baseScore.toFloat().toString()
        } ?: run {
            log.debug { "Could not parse CVSS vector from: '${it.score}'." }
            null to it.score
        }